v23.1/create-changefeed.md

```diff
@@ -157,14 +157,15 @@ Option | Value | Description
 `envelope` | `key_only` / `row`* / `wrapped` | `key_only` emits only the key and no value, which is faster if you only want to know when the key changes.<br><br>`row` emits the row without any additional metadata fields in the message. *You can only use `row` with Kafka sinks or sinkless changefeeds. `row` does not support [`avro` format](#format).<br><br>`wrapped` emits the full message including any metadata fields. See [Responses](changefeed-messages.html#responses) for more detail on message format.<br><br>Default: `envelope=wrapped`
 <a name="format"></a>`format` | `json` / `avro` / `csv`* | Format of the emitted record. For mappings of CockroachDB types to Avro types, [see the table](changefeed-messages.html#avro-types) and detail on [Avro limitations](changefeed-messages.html#avro-limitations). <br><br>*`format=csv` works only in combination with [`initial_scan = 'only'`](#initial-scan). You cannot combine `format=csv` with the [`diff`](#diff-opt) or [`resolved`](#resolved-option) options. Changefeeds use the same CSV format as the [`EXPORT`](export.html) statement. See [Export data with changefeeds](export-data-with-changefeeds.html) for details using these options to create a changefeed as an alternative to `EXPORT`. <br><br>Default: `format=json`.
 <a name="full-table-option"></a>`full_table_name` | N/A | Use fully qualified table name in topics, subjects, schemas, and record output instead of the default table name. This can prevent unintended behavior when the same table name is present in multiple databases.<br><br>**Note:** This option cannot modify existing table names used as topics, subjects, etc., as part of an [`ALTER CHANGEFEED`](alter-changefeed.html) statement. To modify a topic, subject, etc., to use a fully qualified table name, create a new changefeed with this option. <br><br>Example: `CREATE CHANGEFEED FOR foo... WITH full_table_name` will create the topic name `defaultdb.public.foo` instead of `foo`.
+<span class="version-tag">New in v23.1:</span> <a name="gc-protect-expire"></a>`gc_protect_expires_after` | [Duration string](https://pkg.go.dev/time#ParseDuration) | Automatically expires protected timestamp records that are older than the defined duration. In the case where a changefeed job remains paused, `gc_protect_expires_after` will trigger the underlying protected timestamp record to expire and cancel the changefeed job to prevent accumulation of protected data. Use with [`protect_data_from_gc_on_pause`](#protect-pause) to limit the amount of time a changefeed job will remain paused protecting change data.<br><br>See [Garbage collection and changefeeds](changefeed-messages.html#garbage-collection-and-changefeeds) for more detail on protecting changefeed data. 
```

Comment:
oh ok its a Golang duration.  As I mentioned `INTERVAL` in my previous comment, is this also a SQL `INTERVAL` ?  would be nice if so.  

If not then I guess amend my previous comment to say instead of using the text "interval" and linking to `INTERVAL` docs maybe just edit the text referred to in my previous comment to say "duration" with no link

suggest making reference to "protected timestamp" some kind of link to protected timestamps docs.  Esp. since this is an anchor link so folks may jump here directly and not get surrounding context

similarly suggest making the following text links: "change feed job".  Suggest changing "accumulation of protected data" to "accumulation of protected MVCC values" and make MVCC a link to https://www.cockroachlabs.com/docs/v23.1/architecture/storage-layer.html#mvcc